Sette_e_mezzo.py

```diff
@@ -31,20 +31,21 @@
 CardNameLoc      =  (5,5)
 DeckInfoLoc      =  (300,300)
 ScoreLoc         =  (100,300)
-deck             =  pygame.image.load("images/deck.jpg")
 icon             =  pygame.image.load("images/icon.png")
 
 pygame.display.set_icon(icon)
 
 # throwaways (lowercase)
 deal = True
 hand = []
+available = []
 
 # Game loop
 while True:
 
         # available cards 
-        available = [code for code in Cards if not Cards[code].drawn == True]
```

Comment:
This would run every game loop. We can move `available` outside of the game-loop and not have to keep reading this file.